browser/main/modals/PreferencesModal/UiTab.js

```diff
@@ -637,6 +640,28 @@ class UiTab extends React.Component {
               />
             </div>
           </div>
+          <div styleName='group-section'>
+            <div styleName='group-section-label'>
+              {i18n.__('Custom MarkdownLint Rules')}
+            </div>
+            <div styleName='group-section-control'>
+              <div style={{fontFamily}}>
+                <ReactCodeMirror
+                  width='400px'
+                  height='400px'
```

Comment:
Can you make it smaller?

Reply:
I made height of the form smaller from `400px` to `200px`.
![capture](https://user-images.githubusercontent.com/11808736/58360866-fd02ff80-7ec5-11e9-8151-bb627c10a272.PNG)

Reply:
Thx